python-lib/dku_data_processing/format.py

```diff
@@ -69,7 +69,8 @@ def fetch_geo_data(dss_dataset: Dataset, geopoint, detail, tooltips, filters):
         for col_ in filters:
             column_to_retrieves.add(col_['column'])
```

Comment:
typo in `column_to_retrieves`==> `columns_to_retrieve`

Reply:
Fixed in e2ae4af